src/styles/index.scss

```diff
@@ -1,3 +1,64 @@
+@import 'variables';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-width;
+  width: calc($width-day * 7 + 10px * 2 + $gap-width * 6);
+  padding: 10px;
+
+  &__day {
+    position: relative;
+    width: $width-day;
+    height: $height-day;
+    background-color: #eee;
+    border: 1px solid black;
+    cursor: pointer;
+    transition:
+      transform 0.5s,
+      background-color 0.5s;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+        position: absolute;
+        font-size: 30px;
+        color: black;
+        font-family: Arial, Helvetica, sans-serif;
```

Comment:
```suggestion

```
It's better to move common styles to a general class instead of generating the same styling for each 31 element